okapi-core/src/test/java/org/folio/okapi/bean/RoutingEntryTest.java

```diff
@@ -287,4 +286,32 @@ void testMatchUriTenant() {
     t.setPathPattern("/{tenantId}/b/{moduleId}");
     assertTrue(t.matchUriTenant("/diku/b/other", "diku"));
   }
+
+  @Test
+  void testTimerScheduleEncoding() {
+    RoutingEntry t = new RoutingEntry();
+    Assertions.assertThat(t.getDelayMilliSeconds()).isEqualTo(0L);
+    String cron = "3 2 * * *";
+    t.setSchedule(cron);
+    Assertions.assertThat(t.getDelayMilliSeconds()).isGreaterThan(0L);
+    String encode = Json.encode(t);
+    assertEquals(encode, "{\"schedule\":\"" + cron + "\"}");
+    RoutingEntry t2 = Json.decodeValue(encode, RoutingEntry.class);
+    assertEquals(t2.getSchedule(), cron);
+  }
+
+  @Test
+  void testTimerScheduleBadSpecv() {
+    Exception e = assertThrows(DecodeException.class, () ->
+      Json.decodeValue("{\"schedule\":\"3 2 x * *\"}", RoutingEntry.class)
+    );
+    Assertions.assertThat(e.getMessage()).contains("Invalid chars: X");
```

Comment:
Mixing JUnit Jupiter assertions and AssertJ assertions looks odd.
```suggestion
    Assertions.assertThatExceptionOfType(DecodeException.class).isThrownBy(() ->
      Json.decodeValue("{\"schedule\":\"3 2 x * *\"}", RoutingEntry.class)
    ).withMessageContaining("Invalid chars: X");
```

or
```
    Assertions.assertThatThrownBy(() ->
      Json.decodeValue("{\"schedule\":\"3 2 x * *\"}", RoutingEntry.class))
    .isInstanceOf(DecodeException.class)
    .hasMessageContaining("Invalid chars: X");
```

Reply:
ok